packages/entities/entities-routes/src/locales/en.json

```
@@ -190,7 +190,7 @@
         "label": "SNIs",
         "placeholder": "e.g. api.example.com",
         "singular": "SNI",
-        "tooltip": "SNIs that match the route when using stream routing."
+        "tooltip": "SNIs(Server Name Indications) that match the route when using stream routing."
```

nit: Shall we leave a space before the parentheses

```suggestion
        "tooltip": "SNIs (Server Name Indications) that match the route when using stream routing."
```

---

Good catch, space added!